src/main/java/com/github/snuk87/keycloak/kafka/serializer/JsonSerializer.java

```diff
@@ -0,0 +1,38 @@
+package com.github.snuk87.keycloak.kafka.serializer;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.kafka.common.errors.SerializationException;
+import org.apache.kafka.common.serialization.Serializer;
+
+import java.util.Map;
+
+public class JsonSerializer implements Serializer<Object> {
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    public JsonSerializer() {
+
+    }
+
+    @Override
+    public void configure(Map<String, ?> config, boolean isKey) {
+
+    }
+
+    @Override
+    public byte[] serialize(String topic, Object data) {
```

Comment:
What's the difference to `org.apache.kafka.common.serialization.BytesSerializer?`

Reply:
It throws exception on Events and AdminEvents